### PR TITLE
Match most of libkernl/filestub.c

### PR DESCRIPTION
**Pull request by Mc-muffin**
Changed the library compiler from 2.9-990721 to 2.9-991111 that produces better matches for library functions.
Also matched most functions in `libkernl/filestub.c` the missing ones have codegen issues :(

### CODE REVIEW

File: include/lib/libkernl/filestub.h [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+	unsigned char st_mtime[8];
+	unsigned int st_hisize;
+	unsigned int st_private[6];
+} sce_stat;
+
+typedef struct {
+	sce_stat d_stat;
+	char d_name[256];
+	void *d_private;
+} sce_dirent;
```
Comment (Yotona) [inline include/lib/libkernl/filestub.h:28]: Could we include `sifdev.h` rather than redefine these two structs?

File: include/lib/libkernl/filestub.h
Comment (Yotona) [inline include/lib/libkernl/filestub.h:1]: `int` -> `s32`, `unsigned char` -> `u8`, etc. are preferred

**Review by ethteck — APPROVED** [review]
LGTM aside from the suggested changes :)

File: include/lib/libkernl/filestub.h (hunk 1 above)
Comment (Mc-muffin) [reply include/lib/libkernl/filestub.h]: Didn't realize the duplicate

File: include/lib/libkernl/filestub.h
Comment (Mc-muffin) [reply include/lib/libkernl/filestub.h:1]: Done, though I left the function arguments as they appear in `sifdev.h` and left the `char[]` as `char`, do we prefer using `s8` instead?

Comment (Yotona) [reply include/lib/libkernl/filestub.h:1]: Char is fine for arrays and preferred when the usage is unknown

**Review by Yotona — APPROVED** [review]

**Pull request merged**